CRM/Upgrade/Incremental/sql/5.45.alpha1.mysql.tpl

```diff
@@ -1 +1,8 @@
 {* file to handle db changes in 5.45.alpha1 during upgrade *}
+-- Add in missing province for Philippines and update names as per ISO.
+SELECT @PHILIPPINESID := id FROM civicrm_country WHERE name = 'Philippines' AND iso_code='PH';
+INSERT IGNORE INTO civicrm_state_province (country_id, abbreviation, is_active, name) VALUES (@PHILIPPINESID, "DIN", 1, 'Dinagat Islands');
+UPDATE civicrm_state_province SET name = 'Davao del Norte' WHERE country_id = @PHILIPPINESID AND abbreviation = "DAV" AND name = "Davao del Norte";
+UPDATE civicrm_state_province SET name = 'Davao de Oro' WHERE country_id = @PHILIPPINESID AND abbreviation = "COM" AND name = "Davao de Oro";
```

Comment:
Sorry - I bet you thought this would be an easy one, huh? The first two updates are comparing to the new name.

Reply:
Thanks @demeritcowboy fixed now